app/views/providers/visas/edit.html.erb

```diff
@@ -0,0 +1,44 @@
+<%= content_for :page_title, "Visa sponsorship" %>
+<%= content_for :before_content, render_breadcrumbs(:visas) %>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <%= form_with(
+      model: @form_object,
+      url: provider_recruitment_cycle_visas_path(@provider.provider_code, @provider.recruitment_cycle_year),
+      method: :post,
+      scope: "",
+      builder: GOVUKDesignSystemFormBuilder::FormBuilder,
+    ) do |form| %>
+
+      <%= form.govuk_error_summary %>
+
+      <h1 class="govuk-heading-xl">
+        <span class="govuk-caption-xl"><%= @provider.provider_name %></span>
+        Visa sponsorship
+      </h1>
+
+      <%= form.govuk_radio_buttons_fieldset(
+        :request_type,
+        legend: { text: "Can you sponsor Student visas?" },
+        hint: { text: "Applies to fee-paying courses" },
+      ) do %>
+        <%= form.govuk_radio_button :can_sponsor_student_visa, true, label: { text: "Yes" }, link_errors: true %>
+        <%= form.govuk_radio_button :can_sponsor_student_visa, false, label: { text: "No" } %>
+      <% end %>
+
+      <%= form.govuk_radio_buttons_fieldset(
+        :request_type,
+        legend: { text: "Can you sponsor Skilled Worker visas?" },
+        hint: { text: "Applies to salaried courses" },
+      ) do %>
+        <%= form.govuk_radio_button :can_sponsor_skilled_worker_visa, true, label: { text: "Yes" }, link_errors: true %>
+        <%= form.govuk_radio_button :can_sponsor_skilled_worker_visa, false, label: { text: "No" } %>
+      <% end %>
+
+      <%= form.govuk_submit "Save" %>
+    <% end %>
+
+    <p class="govuk-body"><a href="/about-your-organisation">Cancel changes</a></p>
```

Comment:
This link doesn't work. I think we can actually just remove it though, as the "Back" link can be used for cancelling, and having a cancel link close to the Save button can introduce a small risk that users click it by accident.